ir/src/main/scala/org/scalajs/core/ir/Utils.scala

```diff
@@ -66,6 +66,35 @@ object Utils {
     // scalastyle:on return
   }
 
+  def printEscapeJS(str: String, out: java.io.Writer): Unit = {
```

Comment:
Also, how does this performs compared to testing whether any encoding is necessary first, like `escapeJS` does?